chain/network/src/peer_manager/tests.rs

```diff
@@ -0,0 +1,148 @@
+use crate::network_protocol::testonly as data;
+use crate::network_protocol::Encoding;
+use crate::peer;
+use crate::peer_manager;
+use crate::peer_manager::peer_manager_actor::Event as PME;
+use crate::testonly::make_rng;
+use crate::types::{PeerMessage, RoutingTableUpdate};
+use near_logger_utils::init_test_logger;
+use near_network_primitives::time;
+use near_network_primitives::types::NetworkConfig;
+use near_network_primitives::types::{Ping, RoutedMessageBody};
+use near_primitives::network::PeerId;
+use rand::Rng as _;
+use std::collections::HashSet;
+use std::sync::Arc;
+use tokio::net::TcpStream;
+
+// After the initial exchange, all subsequent SyncRoutingTable messages are
+// expected to contain only the diff of the known data.
+#[tokio::test]
+async fn repeated_data_in_sync_routing_table() {
+    init_test_logger();
+    let mut rng = make_rng(921853233);
+    let rng = &mut rng;
+    let mut clock = time::FakeClock::default();
+    let port = crate::test_utils::open_port();
```

Comment:
It seems preexisting, but yeah, this is racy and this is why we should be passing a bound port in.

Reply:
will address in another PR.